shared/templates/static/oval/partition_for_var_tmp.xml

```diff
@@ -0,0 +1,23 @@
+<def-group>
+  <definition class="compliance" id="partition_for_var_tmp" version="1">
```

Comment:
Please move this to `shared/oval`. `shared/templates/static/oval` doesn't exist anymore.